apl-api/src/main/java/com/apollocurrency/aplwallet/api/dto/Currency.java

```diff
@@ -1,14 +1,19 @@
 package com.apollocurrency.aplwallet.api.dto;
 
 
+import com.apollocurrency.aplwallet.api.response.ResponseBase;
+
+import java.util.List;
 import java.util.Objects;
 
-public class Currency {
+public class Currency extends ResponseBase {
```

Comment:
Maybe better to use lombok for getters/setters?

Reply:
Yes its will be better, I use it  for new DTO  .  But, I don't change the classes that are already used in core.